contracts/sol/MACI.sol

```diff
@@ -315,14 +317,11 @@ contract MACI is DomainObjs, ComputeRoot, MACIParameters, VerifyTally {
         Message memory _message,
         PubKey memory _encPubKey
     ) 
-    isAfterSignUpDeadline
```

Comment:
There are still distinct signup and voting periods, but now users can also vote during signup period?

Reply:
Yup, they can.